docs/.build_docs.sh

```diff
@@ -1,3 +0,0 @@
-rm -rf source/generated
```

Comment:
why delete this file?

Reply:
you were using it?
it was duplicate to `make docs` in root